tools/changelogger-release.sh

```diff
@@ -280,10 +280,10 @@ for DS in $( git -c core.quotepath=off diff --name-only projects | sed -E -e 's!
 		debug "  $DS already has an uncommitted change entry file, skipping"
 	elif ! git diff --quiet -- . ":!./composer.lock"; then
 		debug "  $DS has non-lockfile changes"
-		changelogger_add 'Updated package dependencies.'
+		changelogger_add 'Update package dependencies.'
 	else
 		debug "  $DS has lockfile changes only"
-		changelogger_add '' 'Updated composer.lock.'
+		changelogger_add '' 'Update composer.lock.'
```

Comment:
Since you did this one, you should probably do
https://github.com/Automattic/jetpack/blob/080b34119ca27f2b33ccd177bcec27dcbf215d71/tools/check-intra-monorepo-deps.sh#L252
as well

Reply:
1e70618367